webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/AuditsPageActions.js

```diff
@@ -0,0 +1,111 @@
+import history from '../../../history';
+import API from '../../../API';
+import {
+  AUDITS_PATH,
+  AUDITS_API_PATH,
+  AUDITS_PAGE_DATA_RESOLVED,
+  AUDITS_PAGE_DATA_FAILED,
+  AUDITS_PAGE_HIDE_LOADING,
+  AUDITS_PAGE_UPDATE_QUERY,
+  AUDITS_PAGE_CLEAR_ERROR,
+  AUDITS_PAGE_SHOW_LOADING,
+} from '../constants';
+import {
+  selectAuditsSelectedPage,
+  selectAuditsHasError,
+  selectAuditsPerPage,
+  selectAuditsSearch,
+  selectAuditsIsLoadingPage,
+} from './AuditsPageSelectors';
+import { stringifyParams, getParams } from '../../../common/urlHelpers';
+import { translate as __ } from '../../../common/I18n';
+
+// on didMount or popstatee
+export const initializeAudits = () => dispatch => {
+  const params = getParams();
```

Comment:
Shouldn't it suppose to receive the params from the history/location API provided by react-router?

Reply:
we can but we would need the `query-string` npm package ( or another ) to parse it, this avoids adding another package.

Reply:
Is it a technical debt will we need to do eventually?

Reply:
could be, but it would be pretty small